src/main.js

```diff
@@ -112,19 +113,26 @@ function favoriteIdea(event) {
   if (event.target.id === "favoriteButton") {
     var ideaId = event.target.closest("article").id;
 
-    ideas[ideaId].updateIdea();
-    //ideas[ideaId].saveToStorage(ideas[ideaId]);
+  for (var i = 0; i < ideas.length; i++) {
```

Comment:
Same issue as deleteId - we are complicating these functions. If there's a compelling reason we can change it.